packages/effector/index.d.ts

```diff
@@ -155,6 +156,7 @@ export interface Event<Payload> extends Unit<Payload> {
  * The function you can call to trigger an event.
  */
 export interface EventCallable<Payload> extends Event<Payload>, UnitTargetable<Payload> {
+  kind: "event"
```

Comment:
Why we don't go futher and introduce `kind: "event-callable"`?

Reply:
Why do we need to?

So far it is not really necessary, i think - this move was done only to improve TS type inference in a few edge-cases

Reply:
Also there is now a separate runtime field `unit.targetable` 😅

Reply:
I thought more about consistency. Never mind, it looks good for me.